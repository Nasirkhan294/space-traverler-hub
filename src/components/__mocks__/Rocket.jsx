import React from 'react';

const Rocket = () => (
  <div>
    <div className="container">
      <div className="img-container">
        <img src="src/image" alt="something" />
      </div>
      <div className="detail-container">
        <h3>Rocket name</h3>
        <p>
          <span className="badge">
            Reserved
          </span>
        </p>
        <div className="btn-container">
          <button type="button">Reserve Rocket</button>
          <button type="button">Cancel Reservation</button>
        </div>
      </div>

    </div>
  </div>
);

export default Rocket;
