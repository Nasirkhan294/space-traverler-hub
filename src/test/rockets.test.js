import React from 'react';
import renderer from 'react-test-renderer';
import Rocket from '../components/__mocks__/Rocket';

describe('Rockets', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Rocket />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
