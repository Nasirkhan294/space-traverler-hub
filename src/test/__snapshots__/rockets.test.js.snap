// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets renders correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="img-container"
    >
      <img
        alt="something"
        src="src/image"
      />
    </div>
    <div
      className="detail-container"
    >
      <h3>
        Rocket name
      </h3>
      <p>
        <span
          className="badge"
        >
          Reserved
        </span>
      </p>
      <div
        className="btn-container"
      >
        <button
          type="button"
        >
          Reserve Rocket
        </button>
        <button
          type="button"
        >
          Cancel Reservation
        </button>
      </div>
    </div>
  </div>
</div>
`;
